api/src/routing.js

```diff
@@ -167,12 +167,12 @@ app.use(compression({
 //       https://github.com/medic/medic/issues/4089
 
 app.get('/', function(req, res) {
-  if (req.headers.accept === 'application/json') {
-    // couchdb request - let it go
-    proxy.web(req, res);
-  } else {
-    res.sendFile(path.join(extractedResourceDirectory, 'templates/inbox.html'));
-  }
+  res.sendFile(path.join(extractedResourceDirectory, 'templates/inbox.html'));
```

Comment:
Actually on second thought. When app is running vPrev and we deploy v3.5 we temporarily have v3.5's api serving the requests from vPrev. That code won't know to request to `/dbinfo` so it isn't safe to remove the `if (req.headers.accept === 'application/json')` block here ...

Reply:
Nick was eager to test this today - so I pushed that commit into this PR. Sounds like the upgrade is now unblocked.

Reply:
Looks good to me. I think this is ready for AT.